pingdom-checks/veteran-verification.ping

```diff
@@ -25,3 +25,16 @@ pingdom save-check \
   -a group=veteran-verification \
   -a userids_csv="$STATUSPAGE_SANDBOX_VETERAN_VERIFICATION" \
   -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/veteran_verification/v1/health" \
```

Comment:
Is this supposed to be v2?

Reply:
Confirmed this needs to be v2. Please update accordingly. Reference: https://libertyits.slack.com/archives/CBVUWQ21Y/p1665793228244219?thread_ts=1665769935.201359&cid=CBVUWQ21Y